### ISSUE TITLE
`composer show` does not respect --path option if called for single package

### ISSUE THREAD

**Issue by axel-rutz** [composer/composer#7696]
As a shell scripter, i want to extract the path of a package.

When i try this with `composer show --path`,

* `composer show --path foo/*` gives me what i want (name and path)
* `composer show --path foo/bar` does **not** give me what i want (name and path), but seems to ignore the --path option

## Example

```
$ composer show -P twig/*
twig/twig /home/merlin/PhpstormProjects/d8.5-dev/vendor/twig/twig

$ composer show -P symfony/*
symfony/browser-kit             /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/browser-kit
symfony/class-loader            /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/class-loader
symfony/console                 /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/console
symfony/css-selector            /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/css-selector
symfony/debug                   /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/debug
symfony/dependency-injection    /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/dependency-injection
symfony/dom-crawler             /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/dom-crawler
symfony/event-dispatcher        /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/event-dispatcher
symfony/finder                  /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/finder
symfony/http-foundation         /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/http-foundation
symfony/http-kernel             /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/http-kernel
symfony/phpunit-bridge          /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/phpunit-bridge
symfony/polyfill-iconv          /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/polyfill-iconv
symfony/polyfill-mbstring       /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/polyfill-mbstring
symfony/polyfill-php70          /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/polyfill-php70
symfony/process                 /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/process
symfony/psr-http-message-bridge /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/psr-http-message-bridge
symfony/routing                 /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/routing
symfony/serializer              /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/serializer
symfony/translation             /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/translation
symfony/validator               /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/validator
symfony/var-dumper              /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/var-dumper
symfony/yaml                    /home/merlin/PhpstormProjects/d8.5-dev/vendor/symfony/yaml

$ composer show -P symfony/yaml
name     : symfony/yaml
descrip. : Symfony Yaml Component
keywords : 
versions : * v3.4.5
type     : library
license  : MIT License (MIT) (OSI approved) https://spdx.org/licenses/MIT.html#licenseText
source   : [git] https://github.com/symfony/yaml.git 6af42631dcf89e9c616242c900d6c52bd53bd1bb
dist     : [zip] https://api.github.com/repos/symfony/yaml/zipball/6af42631dcf89e9c616242c900d6c52bd53bd1bb 6af42631dcf89e9c616242c900d6c52bd53bd1bb
names    : symfony/yaml
autoload psr-4 Symfony\Component\Yaml\ => . exclude-from-classmap
requires php ^5.5.9|>=7.0.8
requires (dev) symfony/console ~3.4|~4.0
suggests symfony/console For validating YAML files using the lint command
conflicts symfony/console <3.4

```

**Comment by vallode** [comment]
My PR should take care of this issue. I added a path in the verbose package meta output as well as making sure the path flag is respected in a single package call.

**Comment by Sylk** [comment]
Atta' boy @vallode thanks for the fix champ.

**Comment by Seldaek** [comment]
Fixed by #7698
